File: python/word_count.py
from pyspark import SparkContext

sc = SparkContext('spark://master:7077', 'Tutorial')
corpus = sc.textFile("../data/test.txt")

count = corpus.flatMap(lambda line: line.split(" ")).map(
    lambda word: (word, 1)).reduceByKey(lambda a,b: a+b).collect()

print count

File: scala/wordCount.scala
import org.apache.spark.SparkContext

val sc = SparkContext("spark://master:7077", "Parallelize Collection")

val corpus = sc.textFile("../data/pg1342.txt")

val counts = corpus.flatMap(line => line.split(" ")).map(word => (word, 1)).reduceByKey(_+_).collect()

println(counts)


